src/components/QueryOutput/QueryOutput.js

```diff
@@ -1562,10 +1587,16 @@ export class QueryOutput extends React.Component {
   onTableParamsChange = (params, formattedTableParams = {}) => {
     this.tableParams = _cloneDeep(params)
     this.formattedTableParams = formattedTableParams
+    if (this.queryResponse?.data?.data?.fe_req) {
+      try {
+        this.queryResponse.data.data.fe_req.filters = this.formattedTableParams?.filters || []
```

Comment:
Which bug is this fixing? I'm not sure about adding the table filters to the fe_req manually